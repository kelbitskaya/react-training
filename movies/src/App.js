import React from 'react';
import './App.css';
import Image from './components/image';
import Text from './components/text';
import BoldText from './components/boldText';

function App() {
  return (
    <div className="App">
      {React.createElement(
        'h1',
        {className: 'main-title'},
        'Hello World'
      )}
      <Image/>
      <Text/>
      <BoldText/>
    </div>

  );
}

export default App;
