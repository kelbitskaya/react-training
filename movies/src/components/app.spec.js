import React from "react";
import App from "./app";
import rerender from "react-test-renderer";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";

describe("App", () => {
  test("App snapshot", () => {
    const initialStore = {};
    const store = configureStore([])(initialStore);
    const component = rerender.create(
      <Provider store={store}>
        <App />
      </Provider>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
