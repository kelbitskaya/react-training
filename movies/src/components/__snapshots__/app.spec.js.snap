// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot 1`] = `
<h2>
  Something went wrong.
</h2>
`;
